controllers/articles.controllers.js

```diff
@@ -16,3 +20,15 @@ exports.getArticles = (req, res) => {
     res.status(200).send({ articles: rows });
   });
 };
+
+exports.patchArticle = (req, res, next) => {
+  const { inc_votes } = req.body;
+  const { article_id } = req.params;
+  updateArticle(inc_votes, article_id)
```

Comment:
You could use your model that finds an individual article in a Promise.all in this controller to handle the case that the article_id doesn't exist or is invalid as that will already use the error pathways previously set up.